neat/trees/sovtree.py

```diff
@@ -47,46 +47,47 @@ def _setSOV(self, channel_storage, tau_0=0.02):
         self.tau_0      = tau_0 # s
         self.z_a        = self.r_a / (np.pi * self.R_sov**2) # MOhm/cm
         self.g_inf_m    = 1. / (self.z_a * self.lambda_m) # uS
-        # function for SOV approach
```

Comment:
(same question as above)

Reply:
Changes were needed to remove the dependency on `dill`